src/recorderHarMiddleware.ts

```diff
@@ -1,9 +1,9 @@
-import * as http from 'http';
-
 import type express from 'express';
 import type { Entry, Header } from 'har-format';
+import * as http from 'http';
+import zlib from 'node:zlib';
```

Comment:
I'm on latest Node (21), not sure how backwards compatible this type of import is. 

I don't use Node.js on a day-to-day basis so not sure what the best way to import this is.